news/13880-lazy-index

```diff
@@ -0,0 +1,21 @@
+### Enhancements
+
+* Add `conda.core.index.Index` as a faster drop-in replacement of realized dictionary index. (#13880)
+  Note: This removes the implicit initialization of the logging from `get_index`. If you relied on that, you should probably call `conda.gateways.logging.initialize_logging` explicitly.
+
+### Bug fixes
+
+* <news item>
+
+### Deprecations
```

Comment:
I think we have deprecated more things in this PR. Like `get_reduced_index`. Double check.